source/to_cpp1.h

```diff
@@ -1883,12 +1918,13 @@ class cppfront
     //
     auto emit(
         id_expression_node const& n,
-        bool                      is_local_name          = true
+        bool                      is_local_name          = true,
+        bool                      is_class_member_access = false
     )
         -> void
     {   STACKINSTR
         try_emit<id_expression_node::qualified  >(n.id);
-        try_emit<id_expression_node::unqualified>(n.id, false, is_local_name);
+        try_emit<id_expression_node::unqualified>(n.id, false, is_local_name, false, is_class_member_access);
```

Comment:
The first `false` should be `0` now.

Reply:
I see... do you plan to make that change?

Reply:
Yeah.
Temporarily making the parameter `std::same_as<int>` should do the trick.